README.md

```diff
@@ -118,7 +126,7 @@ curl -X 'POST' \
 }'
 ```
 
-- Lastly, obtain a valid BCGov Verified Email credential from the [BCGov Email Verification Service](https://email-verification.vonx.io)
+- Lastly, obtain a Person Credential from the [BC Wallet Showcase](https://digital.gov.bc.ca/digital-trust/showcase) by completing the lawyer demo.
```

Comment:
I messed this up at first because it gives you the lawyer cred first and the person second so I quit after the lawyer one and got confused when the proof didn't work. 😆
It does say "completing the lawyer demo" so probably fine.